src/widget/OktaSignIn.js

```diff
@@ -45,7 +45,7 @@ var OktaSignIn = (function () {
      * @param opts - OAUTH options to refresh the idToken
      */
     function refreshIdToken(idToken, callback, opts) {
-      authClient.refreshIdToken(idToken, opts).then(callback)
+      authClient.idToken.refresh(opts).then(callback)
```

Comment:
I like 3 for now. Let's fix the widget, and file a followup jira to address this in the next major release.